### PR TITLE
docs(headers) change title to "introduction"

### LINKED ISSUE #1054: Some Kong Enterprise docs have repetitive page title and starting header
Some Kong Enterprise docs like https://docs.konghq.com/enterprise/0.34-x/developer-portal/configuration/smtp/ currently show a repetitive (and thus un-helpful) page title and starting header:

dev_portal_smtp_configuration_-_v0_34-x___kong_-_open-source_api_management_and_microservice_management

These pages should have their title and starting header corrected to not be repetitive.

### PR DESCRIPTION
**Pull request by arshadkazmi42**
### Summary

Change 1st subheading of the page to introduction

### Issues resolved

Fix #1054 

- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
I can make all those changes. 
Do we have any list for it? Or I need to manually explore the docs?

**Comment by h3xar0n** [comment]
Thank you, @arshadkazmi42 . Unfortunately, we do not have a list.

**Comment by arshadkazmi42** [comment]
Ok. I will explore through the docs and check if I can get some more of these.

**Comment by arshadkazmi42** [comment]
@h3xar0n I found few more of those, I am a bit confused for this.
https://docs.konghq.com/0.14.x/pdk/kong.client/#kongclient

Do these also need to be changed to `Introduction` title?

**Comment by h3xar0n** [comment]
@arshadkazmi42 yes, please change any initial header to "Introduction". Once you commit, I may reformat a few sections. Thank you for checking.

**Comment by arshadkazmi42** [comment]
@h3xar0n Does it needs to be updated for all old versions also?

**Comment by arshadkazmi42** [comment]
@levfishbluefish which header valuse are you referring to?

**Comment by levfishbluefish** [comment]
Change all 'Introductions' to H2 - depending on the page this may mean there is an H2 under Introduction that needs to be changed to H3

**Comment by arshadkazmi42** [comment]
Ok. got it. Will work on this, and update the PR :+1:

**Comment by arshadkazmi42** [comment]
@levfishbluefish @h3xar0n just a reminder for this PR review

File: app/0.10.x/getting-started/enabling-plugins.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Enabling Plugins
 ---
 
-# Enabling Plugins
+# OveIntroductionrview
```
Comment (h3xar0n) [inline app/0.10.x/getting-started/enabling-plugins.md]:
```suggestion
## Introduction
```

**Comment by h3xar0n** [comment]
Thanks again, @arshadkazmi42 , both for the submission and your patience.

**Comment by arshadkazmi42** [comment]
Thank you for the review and merge :+1:

**Pull request merged**